docs/faq.md

```diff
@@ -25,7 +25,7 @@ with more info.
 Angular can't be found on my page
 ---------------------------------
 
-Protractor supports angular 1.0.6/1.1.4 and higher - check that your version of Angular is upgraded.
+Protractor supports angular 1.5.x and higher - check that your version of Angular is upgraded.
```

Comment:
Not sure we should change this yet - we should do this in an official deprecation and remove the code from clientSideScripts alongside changing the messaging.